tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -290,14 +277,14 @@ public void add(byte[] data) throws IOException {
   /**
    * Adds an element to the end of the queue.
    *
-   * @param data   to copy bytes from
+   * @param data to copy bytes from
    * @param offset to start from in buffer
-   * @param count  number of bytes to copy
-   * @throws IndexOutOfBoundsException if {@code offset < 0} or {@code count < 0}, or if {@code offset + count} is
-   *                                   bigger than the length of {@code buffer}.
+   * @param count number of bytes to copy
+   * @throws IndexOutOfBoundsException if {@code offset < 0} or {@code count < 0}, or if {@code
+   * offset + count} is bigger than the length of {@code buffer}.
    */
   public synchronized void add(byte[] data, int offset, int count) throws IOException {
-    nonNull(data, "buffer");
+    if (data == null) throw new NullPointerException("data == null");
```

Comment:
Not sure where the conventions are from, but this should be made consistent with the other places as:

```
if (data == null) {
  throw new NullPointerException("data == null");
}
```